recipe/post-link.bat

```diff
@@ -1,4 +1,4 @@
 @echo off
-
-"%PREFIX%\Scripts\jupyter.exe" nbextensions_configurator enable --sys-prefix
-if errorlevel 1 exit 1
```

Comment:
Right, I see what you mean, but I was just going for consistency with the bash version (which will exit nonzero on error). If it's inappropriate to exit nonzero, then I guess we ought to alter the bash version to be consistent with the cmd one?